doc/python/hardware.rst

```diff
@@ -88,6 +88,7 @@ The Python wrapper provides an API for:
     .. attribute:: threads
 
         The number of CPU threads available, 1 by default.
+        Passing ``"avail_threads"`` (as string) will query and use the maximum number of threads the system makes available.
```

Comment:
This is actually not implemented.

Reply:
Isn't that what this PR is about? Locally passing `"avail_threads"` gives me the expected number of threads.

Reply:
Yes!